### DISCUSSION TITLE
I suggest new feature - rating of the map

### DISCUSSION THREAD

**Discussion by fheroes2bugs** [ihhub/fheroes2#8781] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

I suggest new feature - rating of the map

Map menu can be wider, we can add new button "rating" and give own rating to every map. Then sort by rating.
Heroes 2 has 1000+ maps so it can be very helpul to sort - if there is no rating, then you did not play it. It the rating is 5/5 then user may be plays this map again.

### Describe the possible solution

Add the button

example:

![image](https://github.com/ihhub/fheroes2/assets/145407420/b74f9276-c270-4662-990c-9b9d95ab2462)

### Additional info

version 1.1.0

**Comment by ihhub** [comment]
This is a game, not a file manager or application store. I am moving this to discussion section.
